examples/move/object_bound/sources/object_bound.move

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module obo::object_bound {
```

Comment:
Could you add a little blurb as a module comment here explaining what this is useful for, and how it would be used?